src/app/shared/forms/planet-markdown-textbox.component.ts

```diff
@@ -11,6 +24,24 @@ import { DialogsImagesComponent } from '../dialogs/dialogs-images.component';
 interface ImageInfo { resourceId: string; filename: string; markdown: string; }
 interface ValueWithImages { text: string; images: ImageInfo[]; }
 
+let easymdeLoader: Promise<void> | null;
+
+function ensureEasyMDELoaded(): Promise<void> {
+  if (!easymdeLoader) {
+    easymdeLoader = import('easymde')
+      .then((module) => {
+        const easyMdeConstructor = module.default ?? module;
+        (globalThis as any).EasyMDE = easyMdeConstructor;
+      })
+      .catch((error) => {
+        easymdeLoader = null;
+        throw error;
```

Comment:
I think just throwing an error here will result in the whole app crashing if easymde doesn't initialize.

Good to catch the error here, but to then just throw an error isn't great. Would be better to display a message that the editor didn't load correctly.